pc/used_ids.h

```diff
@@ -168,8 +168,8 @@ class UsedRtpHeaderExtensionIds : public UsedIds<webrtc::RtpExtension> {
         }
       }
     }
-    RTC_DCHECK(next_extension_id_ >= min_allowed_id_);
-    RTC_DCHECK(next_extension_id_ <= max_allowed_id_);
+    //RTC_DCHECK(next_extension_id_ >= min_allowed_id_);
```

Comment:
Remove.

Reply:
These two lines were commented out in old cg branch only. Instead of removing them, I'm recovering it in this new branch. If we still hit these DCHECK failures, let's see if we can fix it then.